extension/src/popup/components/signTransaction/Operations/index.tsx

```diff
@@ -572,6 +615,13 @@ export const Operations = ({
                   />
                 ) : null}
 
+                {scFunction ? (
+                  <KeyValueWithScValue
+                    operationKey={t("Function")}
+                    operationValue={JSON.stringify(scFunction)}
+                  />
+                ) : null}
+
```

Comment:
Met with Charles and he'll be providing some design tweaks to make this UI nicer